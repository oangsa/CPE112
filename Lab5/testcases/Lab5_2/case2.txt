Enqueue 1
Enqueue 2
Enqueue 3
Enqueue 4
Enqueue 5
Enqueue 6
Peek
Dequeue
Dequeue
Enqueue 20
Peek
Enqueue 30
END
