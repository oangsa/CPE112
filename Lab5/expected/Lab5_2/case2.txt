Queue is full
1 
1 
2 
3 
3 4 5 20 30 
